### ISSUE TITLE
Feature request for ESP32-2432S024C boards.

### ISSUE THREAD

**Issue by BrazenCodebane** [platformio/platform-espressif32#1363]
ISSUE: Implementing Meshtastic on ESP32-2432S024C TFT (commonly known as CYD[Cheap-Yellow-Display]) which uses platformIO. 

board currently unsupported creating a roadblock to the project.

Amazon link to the board:
https://www.amazon.com/gp/product/B0CLGD2DG6/ref=ox_sc_act_title_9?smid=A2RJ79XBQX6W3M&psc=1

**Comment by valeros** [comment]
Hi @BrazenCodebane, usually a variant header file with proper pin definitions in the [Arduino core](https://github.com/espressif/arduino-esp32) is required.

**Comment by Jason2866** [comment]
> board currently unsupported creating a roadblock to the project.

No, just add a custom boards manifest to the project with a folder name `boards`
